app/controllers/application_controller.rb

```diff
@@ -8,6 +8,7 @@ class ApplicationController < ActionController::Base
   rescue_from GdsApi::HTTPGone, with: :error_410
   rescue_from GdsApi::HTTPNotFound, with: :cacheable_404
   rescue_from GdsApi::InvalidUrl, with: :cacheable_404
+  rescue_from GdsApi::HTTPForbidden, with: :error_403
```

Comment:
Nitpick - I'd move this to be between lines 8 and 9, so that the status codes are in ascending order.

Reply:
I considered it, but thought I'd just leave the status codes in descending order, so 503, 410, 404, 403.

Reply:
Makes sense 👍